resteasy-bom/pom.xml

```diff
@@ -40,6 +40,12 @@
                 <groupId>org.jboss.resteasy</groupId>
                 <artifactId>resteasy-jaxb-provider</artifactId>
                 <version>${project.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>javax.xml.bind</groupId>
+                        <artifactId>jaxb-api</artifactId>
+                    </exclusion>
+                </exclusions>

```

Comment:
Why not excluding this for the jaxb-runtime dependency in resteasy-dependencies/pom.xml  ?

Reply:
This is no jaxb-runtime in the BOM.  I checked that first.

Reply:
https://github.com/resteasy/Resteasy/blob/master/resteasy-dependencies-bom/pom.xml#L225 ?

Reply:
@asoldano what is the purpose to have two distinct BOMs anyway? 
https://github.com/resteasy/Resteasy/blob/master/resteasy-bom/pom.xml
https://github.com/resteasy/Resteasy/blob/master/resteasy-dependencies-bom/pom.xml

If used for differrent purposes then they both should handle excludes in the same way, right?